cmd/new.go

```diff
@@ -45,7 +45,7 @@ var newCmd = &cobra.Command{
 			return err
 		}
 
-		if err := writer.Write(f, changelog); err != nil {
+		if err := writer.Write(f, writer.TmplSrcStandard, changelog); err != nil {
```

Comment:
Simple change here - `writer.Write` just takes `standard`.

```suggestion
		if err := writer.Write(f, "standard", changelog); err != nil {
```

Reply:
Doesn't this trade compile time for run-time errors and then it needs more tests to prevent invalid values when it's implicit using types and constants?